MAINTAINERS

```diff
@@ -0,0 +1,12 @@
+# The following is the list of current K3s maintainers
+# Github ID, Name, Email Address
+
+brandond, Brad Davidson, brad.davidson@rancher.com 
```

Comment:
Should these email be the rancher ones or the ones associated with the github account?

Reply:
Github uses the CODEOWNERS file for automation, I think this is more just a documentation thing? @cjellick used all of our rancher.com addresses when creating this file in k3s.

Reply:
this is what i did in k3s...and i just yanked all the rancher addresses. It doesn't really matter from any official capacity. I would say, if someone wants to change theirs, they can. But for the purposes of getting this merged, I'm just going to roll with this